cypress/e2e/views/taskmanager.view.ts

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+export const tasksMenuName = "Task Manager";
```

Comment:
Line 17 is not for this file, view file is for selectors only. We have `constants` file for such strings.

Reply:
I see, thanks for pointing this out. I will move it